pages/spaces.jsx

```diff
@@ -0,0 +1,241 @@
+    icon2xUrl: "/assets/images/space-icon/Localization@2x.png",
+    iconAlt: "Localization icon",
+    name: "Localization",
+    description:
+      "<p>All languages and cultures are uniquely different and valuable — and they should be treated this way on the Web, too. In order for the Web to empower everyone, it must be adapted to diverse communities and peoples. The next billion people to come online shouldn’t find an English-only Web.</p>" +
+      "<p>In the Localization Space, participants learn language localization skills and nuances, how to include diversity in products, and how to bring this philosophy back to their local communities.</p>",
+    wranglers: [
+      {
+        name: "Heather Bailey"
+      },
+      {
+        name: "Dwayne Bailey"
+      }
+    ]
+  }
+];
+
+var Space = React.createClass({
+  render: function() {
+    return (
+      <div className="space-card">
+        <div className="detail">
+          <div className="header">
+            <div className="img-container">
+              <ImageTag src1x={this.props.iconUrl}
+                        src2x={this.props.icon2xUrl}
+                        alt={this.props.iconAlt}/>
+            </div>
+            <h1>{this.props.name}</h1>
+          </div>
+          <div className="description" dangerouslySetInnerHTML={{__html: this.props.description}}></div>
+        </div>
+        <div className="wranglers">
+          <h2>Wranglers:</h2>
+          <ul>
+          {
+            this.props.wranglers.map(function(wrangler,i) {
+              return (
+                <li key={i}>
+                  <div className="name">{wrangler.name}</div>
+                  { wrangler.twitter ? <div className="twitter"><a href={"https://twitter.com/"+wrangler.twitter}>{wrangler.twitter}</a></div> : null }
+                </li>
+              )
+            })
+          }
+          </ul>
+        </div>
+      </div>
+    );
+  }
+});
+
+
+var SpacesPage = React.createClass({
+  spaces: SpacesInfo,
+  render: function() {
+    return (
+      <div className="volunteer-page">
+        <Header/>
+        <HeroUnit image="/assets/images/spaces.png"
+                  image2x="/assets/images/spaces.jpg">
+          Spaces
+        </HeroUnit>
+        <div className="white-background">
+          <div className="content centered wide">
+            <h1 className="expect-header">How the MozFest Spaces Work</h1>
+            <p>To help you get the most out of MozFest, we organize the program into two formats: Spaces and Pathways.</p>
+            <p>Spaces are physical hubs, typically located on a single floor in the venue, that explore a big theme throughout the festival.</p>
+            <p>Pathways are series of sessions, often appearing across several floors, that are tailored for participants to learn a set of skills.</p>
+            <div className="horizontal-rule"></div>
+          </div>
+        </div>
+        {
+           this.spaces.map(function(space,i){
```

Comment:
this will run the map each time `render()` is called, but the original content is static, so the map will be static too. We can refactor this so that the map is computed on `componentWillMount`, once, so that we can simply use the result directly in `render`:
```
...
componentWillMount: function() {
  this.spaces = this.spaces.map(function(space,i) {
    var whiteBg = (i%2==0) ? "white-background" : "";
    return (
      <div className={whiteBg} key={i}>
        <div className="content wide">
          <Space {...space} />
        </div>
      </div>
    );
  };
},

render: function() {
  ...
}
}